webapp/mapTest4.html

```diff
@@ -342,16 +342,87 @@
               .addTo(map); 
               }
               
-              function pressBtn9(){
-              var u = document.getElementById('button_press9').value;
-			  map.flyTo({center: [-122.284331,37.518310],speed: 0.3});
+              function pressBtnA(){
```

Comment:
It would be a best practice to use more descriptive function names.  This would make the code much more readable and self documenting. pressBtnXXXX is good, but I have no idea which button 'A' is.